src/services/auth/plugins/password/service.ts

```diff
@@ -40,6 +66,104 @@ export class MemberPasswordService {
     await memberPasswordRepository.patch(actor.id, newPassword);
   }
 
+  /**
+   * Modify the password of a member. Force the change without checking the current password.
+   * Check if the token is registered in the redis database. If it is, delete it and change the password of the associated member.
+   * @param repositories Object with the repositories needed to interact with the database. Must contain a memberPasswordRepository.
+   * @param password New password.
+   * @param token The JSON Web Token associated to the member that wants to reset the password.
+   * @returns void
+   */
+  async forcePatch(repositories: Repositories, password: string, token: string): Promise<void> {
+    const id = await this.redis.get(`${REDIS_PREFIX}${token}`);
+    if (!id) return;
+    await this.redis.del(token);
+    const { memberPasswordRepository } = repositories;
+    await memberPasswordRepository.patch(id, password);
+  }
+
+  /**
+   * Create a password reset request.
+   * Push a JSON Web Token to the redis database with the member id as value.
+   * If the email is not registered, do nothing. If the member doesn't have a password, do nothing.
+   * The token will be valid for 24h, after that it will be deleted by the redis database.
+   * The token contains an empty payload, it is only used to identify the member.
+   * @param repositories Object with the repositories needed to interact with the database. Must contain a memberRepository and a memberPasswordRepository.
+   * @param email The email of the member that requested the password reset.
+   * @returns The JSON Web Token to reset the password and the language of the member. Otherwise, undefined if the email is not registered or the member doesn't have a password.
+   */
+  async createResetPasswordRequest(
+    repositories: Repositories,
+    email: string,
+  ): Promise<{ token: string; member: Member } | undefined> {
+    const { memberRepository } = repositories;
+    const member: Member = await memberRepository.getByEmail(email);
+
+    if (!member) return;
+    if (!(await MemberPasswordRepository.getForMemberId(member.id))) return;
```

Comment:
Same as above, always use braces for one-line conditionals. This improves clarity and code refactoring.